addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -26,7 +26,7 @@ const GalleryWidget = publicWidget.Widget.extend({
      * @param {Event} ev
```

Comment:
Commit: 
- The title should start with a verb :wink: (=> suggestion: [FIX] website: fix "Images Wall" slideshow with link on image)
-  > by clicking an image from the Images wall snippet wich is a link.
    
   ->  I would rather say "when clicking on an "Images Wall" snippet image that has been set as a link."  :)

- > Drag and drop an "Images wall"

    -> Nitpicking: "Images Wall" with a capital "w" :stuck_out_tongue: